### ISSUE TITLE
EC2 get_transit_gateway_route_table_associations API not supported

### ISSUE THREAD

**Issue by kbalk** [spulec/moto#4130]
I know there's been a lot of changes related to transit gateway recently and was wondering if there were plans to implement the "get_transit_gateway*" APIs, and in particular `get_transit_gateway_route_table_associations`.

**Comment by bblommers** [comment]
@whummer Do you have any plans to implement this soon?

**Comment by macnev2013** [comment]
Hi @kbalk @bblommers , We have added support for `get_transit_gateway_route_table_associations` and many more in [PR 4089](https://github.com/spulec/moto/pull/4089).
Thanks.

**Comment by kbalk** [comment]
@macnev2013 @bblomers I apologize ... I pulled the latest moto, but I was running localstack, which doesn't have the changes yet.  I just checked the moto code and yes, it's there ... I should have done that first.  I'm so sorry.  I'm learning!

I was having problems with a VPN Gateway and it's probably the same thing ... localstack and not the latest moto.

I was trying to track down how moto was pulled in by localstack and it's a bit confusing.  I think it also depends on whether you're using the docker image of localstack.

**Comment by kbalk** [comment]
@macnev2013 @bblommers I modified my test environment to use moto for ec2 services and reran my Terraform integration tests.  I have 2 that are now failing because `disassociate_transit_gateway_route_table` isn't implemented.  I checked the moto code and I don't see logic for it, so I don't think I goofed up there.

The other 2 vpc-related Terraform scripts I'm dealing with are failing with the error:

```
  File "/home/kbalk/plus3it/moto/moto/ec2/responses/transit_gateway_attachments.py", line 104, in disable_transit_gateway_route_table_propagation
    transit_gateway_propagation = self.ec2_backend.disable_transit_gateway_route_table_propagation(
  File "/home/kbalk/plus3it/moto/moto/ec2/models.py", line 6708, in disable_transit_gateway_route_table_propagation
    self.transit_gateway_propagations[
KeyError: 'tgw-attach-62b306a9ea918cb00'
```

I don't know if you're interested in fixing these errors, if I need to create new tickets for them and if it would help to have the Terraform for them.  It takes a bit of work to get the Terraform winnowed down for a nice test case, but if it would help for the second problem, I could create a test file for you.
